README.md

```diff
@@ -50,6 +50,19 @@ ZCBOR_STATE_E(encode_state, n, payload, payload_len, 0);
 
 The CBOR libraries assume little-endianness by default, but you can define ZCBOR_BIG_ENDIAN to change this.
 
+Configuration
+-------------
+
+The C library has a few compile-time configuration options which can be enabled by adding them as compile definitions to the build.
+
+Name                      | Description
+------------------------- | -----------
+`ZCBOR_CANONICAL`         | When encoding lists and maps, do not use indefinite length encoding. This increases code size and makes the encoding library more often use state backups.
```

Comment:
`This increases code size...` <- is "This" referring to this `ZCBOR_CANONICAL` or using indefinite length encoding? Maybe we could clarify.

Reply:
`ZCBOR_CANONICAL`. Updated the text.